src/main/java/tdl/s3/cli/ProgressBar.java

```diff
@@ -0,0 +1,39 @@
+package tdl.s3.cli;
+
+import tdl.s3.sync.SyncProgressListener;
+
+public class ProgressBar implements SyncProgressListener {
+
+    private final int width = 100;
+
+    private int totalSizeBytes = 0;
+
+    public void updateProgress(double percentage) {
+        System.out.print("\r[");
+        int i = 0;
+        for (; i <= (int) (percentage * width); i++) {
+            System.out.print(".");
+        }
+        for (; i < width; i++) {
+            System.out.print(" ");
+        }
+        System.out.print("]");
```

Comment:
Given that the command line tool is mostly used for testing, it will be useful to display all the data we hold. Example:
[...    ] Progress 124/12412. Uploading 4 parts ( 2 files ).